doc/source/changes/version_0_29.rst.inc

```diff
@@ -13,6 +13,51 @@ Backward incompatible changes
 New features
 ------------
 
+* added new module to create arrays with values generated randomly following a few different distributions, or shuffle
+  an existing array along an axis:
+
+    >>> from larray.random import *
+
+    generate integers between two bounds (0 and 10 in this example)
+
+    >>> randint(0, 10, axes='a=a0..a2')
+    a                   a0                  a1                  a2
+       0.33293756929238394  0.5331412592583252  0.6748786766763107
```

Comment:
values are not supposed to be integers with randint?

Reply:
oops :)